CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 1.7.1 Release candidate
+
+### Breaking changes
+
+### New features
+
+### Improvements
+
+### Deprecated
+
+### Bug fixes
+
+- Fix the "email all users" feature by setting the _To_ email to the logged user's email [#8343](https://github.com/opencrvs/opencrvs-core/issues/8343)
+
```

Comment:
As we are including this in v1.7.0, could you remove this block and move the changelog to the 1.7 section please?